src/cache/mod.rs

```diff
@@ -33,8 +33,11 @@
 use std::collections::{hash_map::Entry, HashMap, HashSet, VecDeque};
 use std::default::Default;
 use std::str::FromStr;
+use std::time::Duration;
 
 use async_trait::async_trait;
+#[cfg(feature = "cache")]
```

Comment:
The entire cache module is gated behind the cache feature so this cfg isn't needed